### ISSUE TITLE
manual/javaGuide/main/inject/JavaInjection.md

### ISSUE THREAD

**Issue by garbagetown** [playframework-ja/translation-project#959]
- https://github.com/playframework-ja/translation-project/tree/2.3.5/manual/javaGuide/main/inject/JavaInjection.md
- diffs between [2.2.0/manual/javaGuide/main/inject/JavaInjection.md](https://github.com/playframework-ja/translation-project/tree/2.2.0/manual/javaGuide/main/inject/JavaInjection.md) and [2.3.5/manual/javaGuide/main/inject/JavaInjection.md](https://github.com/playframework-ja/translation-project/tree/2.3.5/manual/javaGuide/main/inject/JavaInjection.md).

```
--- /Users/garbagetown/Desktop/2.2.0/manual/javaGuide/main/inject/JavaInjection.md  2013-07-29 23:52:24.000000000 +0900
+++ //Users/garbagetown/Desktop/2.3.5/manual/javaGuide/main/inject/JavaInjection.md 2014-07-28 12:26:38.000000000 +0900
@@ -1,3 +1,4 @@
+<!--- Copyright (C) 2009-2013 Typesafe Inc. <http://www.typesafe.com> -->
 By default Play binds URLs to controller methods _statically_, that is, Controller instances are created by the framework and then the appropriate static method is invoked depending on the given URL. In certain situations, however, you may want to manage controller creation and that's when the new routing syntax comes handy. 

 Route definitions starting with '''@''' will be managed by '''play.GlobalSettings#getControllerInstance''' method, so given the following route definition: 

```

**Comment by garbagetown** [comment]
The external link at the last row is doubled. This is the known bug of translation copier.

**Comment by garbagetown** [comment]
:hand:

**Comment by garbagetown** [comment]
merged at #1080
